test/debug_naming_tests.cc

```diff
@@ -0,0 +1,65 @@
+#include "task.h"
+#include "task_manager.h"
+#include "types.h"
+
+#include <catch2/catch_template_test_macros.hpp>
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_string.hpp>
+
+#include <celerity.h>
+
+#include "test_utils.h"
+
+namespace celerity {
+namespace detail {
+	TEST_CASE_METHOD(test_utils::runtime_fixture, "tasks debug names", "[debug]") {
```

Comment:
A more descriptive test case name like "debug names can be set and retrieved from tasks" could improve this!

Reply:
you're right